src/components/Footer/Footer.jsx

```diff
@@ -25,7 +25,7 @@ const Footer = () => (
       <section className="footer__right">
         <Link className="footer__link" to="/glossary/">Glossary</Link>
         <Link className="footer__link" to="/branding/">Branding</Link>
-        <Link className="footer__link" to="//gitter.im/webpack/webpack">Gitter</Link>
+        <Link className="footer__link" to="https://gitter.im/webpack/webpack">Gitter</Link>
```

Comment:
I was going to commend about this https://www.paulirish.com/2010/the-protocol-relative-url/, but then realize is from 8 years ago and see that is now considered an antipattern, also realize how old I am 📟

Reply:
I very much know that feeling :D